actions/notify-slack/get-info.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+function get-info() {
+  set -eo pipefail
+
+  if [ -z "${EMOJI:-}" ]; then
+
+    # MESSAGE="A $TYPE for $APPLICATION:$VERSION to $ENVIRONMENT resulted in $STATUS"
+    case "${STATUS:-}" in
+    success)
+      EMOJI=large_green_circle
+      ;;
+    failure)
+      EMOJI=red_circle
+      ;;
+    started)
+      EMOJI=large_blue_circle
+      # MESSAGE="A $TYPE has been started for $APPLICATION:$VERSION to $ENVIRONMENT"
+      ;;
+    *)
+      EMOJI=white_circle
+      ;;
+    esac
+
+  fi
+
+  if [ -z "${MESSAGE:-}" ]; then
+    MESSAGE="A ${TYPE:-} for ${APPLICATION:-}:${VERSION:-} to ${ENVIRONMENT:-} resulted in ${STATUS:-}"
+  fi
+  echo emoji="$EMOJI" >>"$GITHUB_OUTPUT"
+  echo "message=${MESSAGE:-}" >>"$GITHUB_OUTPUT"
+  TIMESTAMP="$(date +%s)"
+  echo "timestamp=$TIMESTAMP" >>"$GITHUB_OUTPUT"
```

Comment:
```suggestion
    esac
  fi
  echo emoji="$EMOJI" >>"$GITHUB_OUTPUT"

  if [ -z "${MESSAGE:-}" ]; then
    MESSAGE="A ${TYPE} for ${APPLICATION}:${VERSION} to ${ENVIRONMENT} resulted in ${STATUS}"

    if [ "$STATUS" = started ]; then
      MESSAGE="A $TYPE has been started for $APPLICATION:$VERSION to $ENVIRONMENT"
    fi
  fi
  echo "message=${MESSAGE:-}" >>"$GITHUB_OUTPUT"

  TIMESTAMP="$(date +%s)"
  echo "timestamp=$TIMESTAMP" >>"$GITHUB_OUTPUT"
```


Let's please keep sections of code together that related and space them out to group them.

I have also added the MESSAGE change when STATUS is started.

Please do not default required inputs such as TYPE, APPLICATION, ENVIRONMENT, VERSION, etc.